src/uu/sleep/src/sleep.rs

```diff
@@ -61,15 +61,20 @@ pub fn uu_app() -> Command {
 
 fn sleep(args: &[&str]) -> UResult<()> {
     let mut arg_error = false;
-    let intervals = args.iter().map(|s| match uucore::parse_time::from_str(s) {
-        Ok(result) => result,
-        Err(err) => {
-            arg_error = true;
-            show!(USimpleError::new(1, err));
-            Duration::new(0, 0)
-        }
-    });
-    let sleep_dur = intervals.fold(Duration::new(0, 0), |acc, n| acc + n);
+    let sleep_dur = args
+        .iter()
+        .filter_map(|input| {
+            uucore::parse_time::from_str(input.trim()).ok().or_else(|| {
+                arg_error = true;
+                show!(USimpleError::new(
+                    1,
+                    format!("invalid time interval '{input}'")
+                ));
```

Comment:
This is good, but since we're already keeping track of `arg_error`, let's simplify by just printing:
```
show_error!("invalid time interval '{input}'");
```

Normally, I'd encourage `USimpleError`, but in think case I think it's clearer to leave it out, so that it's clear where the exit code is set.